WooCommerce/Classes/Extensions/UIImage+Dot.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+extension UIImage {
+    /// Draws a dot at the top right corner of a new image along with the original image.
```

Comment:
Nice! This is what we should probably have done with the tabbar icon for the notifications